packages/dtslint/src/index.ts

```diff
@@ -170,6 +180,42 @@ async function runTests(
       await testTypesVersion(versionPath, low, hi, expectOnly, undefined, isLatest);
     }
   }
+
+  if (!packageJson.nonNpm && !expectOnly && !noAttw) {
+    const attwJson = joinPaths(dtRoot, "attw.json");
```

Comment:
we're going to want to config npm-naming/dts-critic/expect as well -- should this be called dtslint.json or something?

Reply:
At one point I think we decided this was a good idea, but I feel like we’ve since moved toward thinking none of these things should be configurable/disableable:

- npm naming: specifying `"nonNpm": "conflict"` is the only user-activated exemption mechanism; we may need an “already failing, ignore” list but that could be hidden inside DT-tools(?)
- dts-critic: end goal is suggestions-only, no need to configure/disable
- expect: no need to configure/disable